spec/controllers/projects_controller_spec.rb

```diff
@@ -56,6 +61,7 @@
       end
 
       it { is_expected.to render_template(:new) }
+
```

Comment:
Be careful with blank spaces and lines. (But a blank line at the end of the file is desirable)